ProjFS.Mac/PrjFSKextTests/ProviderMessagingMock.cpp

```diff
@@ -49,7 +71,15 @@ bool ProviderMessaging_TrySendRequestAndWaitForResponse(
         procname,
         kauthResult,
         kauthError);
-    return true;
+    
+    if (s_cleanupRoots)
+    {
+        VirtualizationRoots_Cleanup();
+    }
+    
+    s_requestCount++;
+    
+    return s_requestResult && !(s_secondRequestResult == true && s_requestCount == 2);
```

Comment:
Why does `s_requestResult` impact the what is returned when `s_requestCount` is 2?

Based on how it's used, would it be accurate to rename `s_requestResult` (and its setter) to include "default" in the name, e.g. `s_defaultRequestResult`, and then update this code to:

```
if (s_requestCount == 2)
{
    return s_secondRequestResult;
}
else
{
    return s_defaultRequestResult;
}